addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -16,7 +16,7 @@ $MAX-Z-INDEX : 2147483647 !default;
 }
 
 .oe_slides_panel_footer {
-    background-color: $light;
+    width: 100%;
```

Comment:
Why removing `container-fluid` on this element to add back `width: 100%;` here ?